package.json

```diff
@@ -24,15 +24,15 @@
     "eslint": "^7.32.0",
     "eslint-plugin-square": "^20.0.2",
     "jsdom": "^22.1.0",
-    "svelte": "^3.0.0",
+    "svelte": "^4.1.2",
     "typescript": "^4.5.5",
     "vitest": "^0.34.1"
   },
   "dependencies": {
     "cookie-storage": "^6.1.0"
   },
   "peerDependencies": {
-    "svelte": "^3.0.0"
+    "svelte": "^4.1.2"
```

Comment:
This would block consumers from using this unless they had over 4.1.2 
Instead we want to use semantic versioning to declare a range of compatible svelte versions from 3 to 4 
https://nodejs.org/es/blog/npm/peer-dependencies

Reply:
Good catch! I updated this with a range from 3.0.0 > 5.0.0 https://github.com/square/svelte-store/pull/76/commits/bfe52ea7748503c59b7959675cd8e7138577788d